Windows/Main.java

```diff
@@ -135,16 +144,43 @@ public static void selectAnimal() {
         secondInput = true;
       }
     }
-    // if true, it will hide the panel, and create a new panel that shows the results of the
+    if(user.equalsIgnoreCase("custom") || user2.equalsIgnoreCase("custom")) {
+        Custom customAnimal = new Custom();
```

Comment:
I thought you changed all this?

Reply:
Yes, I did change this, I haven't pushed my current changes yet.